### ISSUE TITLE
Vector Summation accumulator

### ISSUE THREAD

**Issue by dvadym** [OpenMined/PipelineDP#43]
## Context

Currently the [Accumulator Framework](https://github.com/OpenMined/PipelineDP/issues/37) development is in progress. 
The accumulator is the object that accumulates values and produces some the aggregated result. The computing of all Differential Private (DP) metrics will be done in corresponding accumulators.

## Goals

To implement class `VectorSummationAccumulator` which computes a Differential Private sum of input vectors of the same size. 
`VectorSummationAccumulator` should be a sub-class of [Accumulator](https://github.com/OpenMined/PipelineDP/blob/5f8e2b5e1265af6e29f3c2fb79d0c9846a63ba43/pipeline_dp/accumulator.py#L12) class.

### Example of aggregations

The input is [(1, 0, 0), (0, 1, 0), (0, 1, 0), (0, 0, 1)].

The output should be (1, 2, 1) + the random noise vector

Note, that it's possible to compute histograms using vector summation primitive.

## API

```
@dataclass
class VectorSummationParams:
   noise:NoiseKind
   max_norm:float


class VectorSummationAccumulator(Accumulator): 
    def __init__(params:VectorSummationParams, values: Iterable[Tuple[double]])
    def add_value(self, value:Tuple[double])  # returns self
    def add_accumulator(self, accumulator: 'VectorSummationAccumulator')  # returns self
    def compute_metrics(self) -> Tuple[double]
```

## Stages of the development

1. Implement vector summation computation.
2. Add anonymization to output (this is better to do in a separate PR)

**Comment by lovroprepolec** [comment]
I am interested in implementing this feature.

**Comment by dvadym** [comment]
Sure, go ahead! Thanks!

**Comment by dvadym** [comment]
FYI: API of the class was updated (to represent the current development in accumulator framework)

**Comment by lovroprepolec** [comment]
Ack. Thx.

**Comment by levzlotnik** [comment]
Hey, I'd like to take this

**Comment by dvadym** [comment]
Sure, thank you!

**Comment by levzlotnik** [comment]
Hey, this was implemented in #67, we can close this issue now.

**Comment by dvadym** [comment]
levzlotnik@, could you please do plumbing for Vector summation in order to be able to run Vector Summation E2E?

[Here](https://github.com/OpenMined/PipelineDP/pull/87/files) (look at the first change in accumulator.py) is an example of doing this for sum
